src/modules/coreHttpApi/openapi.yml

```diff
@@ -3833,6 +3849,16 @@ components:
         - id
         - secretKey
 
+    FileReferenceTruncated:
+      type: object
+      properties:
+        reference:
+          type: string
+          nullable: false
+          description: The base64 encoded truncated reference of the File or the reference of a token containing the reference to a File.
```

Comment:
Hm, müsste POST /api/v1/Files/Peer nicht FileReference | FileReferenceTruncated | TokenReferenceTruncated akzeptieren? Dementsprechend müsste dann auch die Description hier angepasst werden.

Reply:
Ehm, genau das steht doch in der description?!